.changeset/grumpy-games-fix.md

```diff
@@ -0,0 +1,7 @@
+---
+"@siteimprove/alfa-rules": minor
+---
+
+**Fixed:** The function `getClickableBox` now only considers visible descendants.
+
+The return type of the function was also changed from `Option` to `Result`.
```

Comment:
```suggestion
---
"@siteimprove/alfa-rules": patch
---

**Fixed:** SIA-R111 and SIA-R113 now ignores invisible descendants of interactive elements when computing the clickable areas.
```
* Since we do not alter any public API (just an internal one), this is a `patch` bump (even if the `option` => `Result` change is not backward-compatible). If we want to emphasise that change (e.g. for our internal consumers), that would deserve a separate breaking changeset; but here it should be OK since nobody else than Alfa is using this function.
* Since the actual change is not really exposed to users, the changeset should focus on the visible fix, i.e. the improvements in the rules.